frontend/src/components/public_group/PublicGroupWhoWeAre.js

```diff
@@ -40,6 +41,10 @@ export default class PublicGroupWhoWeAre extends React.Component {
               </div>
             </div>
           ) : null}
+
+          {isSupercollective ?
+            <RelatedGroups title={''} groupList={group.superCollectiveData} {...this.props} />
```

Comment:
That's where the naming breaks. `superCollectiveData` is confusing in this context. It should really be `group.subGroups` or `group.linkedGroups`.